addons/sale/static/src/js/product_list/product_list.scss

```diff
@@ -0,0 +1,25 @@
+.quantity_alignment {
+    @include media-breakpoint-down(md) {
+        text-align: end;
+    }
+
+    
+    @include media-breakpoint-up(md) {
+        text-align: center;
+    }
+}
+
+table.o_sale_product_configurator_table > tbody > tr > td > td {
+    min-width: 50px;
+}
+
+@include media-breakpoint-down(md) {
+    .fixed_border_for_block_display {
```

Comment:
Could you use a clearer class name or select the element without adding a class?